spec.md

```diff
@@ -3326,9 +3355,13 @@ str(1)                          # '1'
 str("x")                        # 'x'
 str([1, "x"])                   # '[1, "x"]'
 str(0.0)                        # '0.0'        (formatted as if by "%g")
-str(b"abc")                     # 'b"abc"'
+str(b"abc")                     # 'abc'
 ```
 
+The string form of a bytes value is the UTF-K decoding of the bytes.
+Each byte that is not part of a valid encoding is replaced by the
```

Comment:
Same issue as above regarding the replacement algorithm.

Reply:
Yep, same answer.